src/main/java/seedu/address/model/hotel/Room.java

```diff
@@ -11,13 +12,15 @@
     private int roomNumber;
     private boolean isOccupied;
     private Person person;
-
+    private Task task;
```

Comment:
Shouldn't `Room` contain an `ArrayList<Task>`?

Reply:
Okay, leave this part to me. I will add in the `TaskList`.

Reply:
I'll change it if you want.